src/scripts/install.py

```diff
@@ -363,20 +382,26 @@ def setup_authentication(jira_password, region, pk_passphrase=None):
     return private_key, pk_passphrase, jira_password, rsa_public_key
 
 
-def gen_envs(jira_user, jira_project, jira_url, jira_password, account, region,
+def gen_envs(jira_user, jira_project, jira_url, jira_password, account, region, host, port, protocol,
              private_key, pk_passphrase, aws_key, aws_secret, **x):
+    # for the .envs file being written, we redesigned the format,
+    # so that currently, host become the TOP argument
```

Comment:
what are you trying to clarify with this comment? older versions of the format don't seem pertinent to someone reading this code